splitter/splitter.go

```diff
@@ -251,6 +276,7 @@ func (s *Splitter) StartWithListener(listen net.Listener) error {
 	e.GET("/xrpc/_health", s.HandleHealthCheck)
 	e.GET("/_health", s.HandleHealthCheck)
 	e.GET("/", s.HandleHomeMessage)
+	e.POST("/v1/requestRequestCrawl", s.RequestRequestCrawlHandler)
```

Comment:
feels like we should require auth (admin auth) on this endpoint.

eg, possible security concerns:
- URL isn't sanitized, so could be internal URLs (eg, 10.0.0.1), resulting in SSRF problems
- there are no rate-limits, so a bad actor could create huge numbers of clients, consuming both rainbow resources and request load on target services
- loop detection

Reply:
we should definitely have an auth token; I was assuming intra-datacenter usage with a frontend router that would only pipe `subscribeRepos` in from the outside, but auth token is good